### PR TITLE
magento/magento2#: Magento_Config module. Fix content in read file

### PR DESCRIPTION
**Pull request by atwixfirster**
### Description (*)

Provides fix for the DevDocs issue - https://github.com/magento/devdocs/issues/5246

### Fixed Issues (if relevant)

1. magento/devdocs#5246: Markdown linting: No space after hash on atx style header (MD018 )

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

Thank you!

CC: @dshevtsov

### CODE REVIEW

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-5751](https://jira.corp.magento.com/browse/ENGCOM-5751) has been created to process this Pull Request
:eight_spoked_asterisk: @dmytro-ch, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
QA not applicable

File: app/code/Magento/Config/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-#Config
+# Magento_Config module
+
+## Config
```
Comment (dshevtsov) [inline app/code/Magento/Config/README.md]: Remove the heading as this is a duplicate of the level 1 heading.

**Review by dshevtsov — CHANGES_REQUESTED** [review]

File: app/code/Magento/Config/README.md (hunk 1 above)
Comment (atwixfirster) [reply app/code/Magento/Config/README.md]: **done**

File: app/code/Magento/Config/README.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
-#Config
+## Config
```
Comment (dshevtsov) [inline app/code/Magento/Config/README.md]:
```suggestion
# Magento_Config module
```

**Review by dshevtsov — CHANGES_REQUESTED** [review]

File: app/code/Magento/Config/README.md (hunk 2 above)
Comment (atwixfirster) [reply app/code/Magento/Config/README.md]: **done**

**Review by dshevtsov — APPROVED** [review]

**Pull request merged**